src/components/about/About.js

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import "./about.scss"
+
+const About = props => {
+  return (
+    <div className="outer-container">
+      <div className="about-container">
+        <h1>
+          <span id="about-us-header">ABOUT US</span>
+        </h1>
+        <p>
+          For entry level developers, people interested in starting a new career
+          in tech, and anyone wanting to encourage and help newer developers.
+          Join us to hear relevant speakers, have a slice of pizza, and make new
+          connections! We look forward to meeting you. If you’d like to be a
+          part of our slack channel, you can be a part of the conversation here:
+          http://bit.ly/JrDevSlackInvite
```

Comment:
This looks like a URL. Let's make it a link and put the "HT" back in HTML!

Reply:
Created `<a href>` around "conversation here" in place of URL